frontend/src/features/mapSideBar/SidebarContents/subforms/BuildingReviewPage.test.tsx

```diff
@@ -0,0 +1,164 @@
+import React from 'react';
+import { fireEvent, render } from '@testing-library/react';
+import { noop } from 'lodash';
+import { Formik } from 'formik';
+import pretty from 'pretty';
+import { BuildingReviewPage } from './BuildingReviewPage';
+import configureMockStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
+import { ILookupCode } from 'actions/lookupActions';
+import * as API from 'constants/API';
+import * as reducerTypes from 'constants/reducerTypes';
+import { Provider } from 'react-redux';
+
+const mockStore = configureMockStore([thunk]);
+const lCodes = {
+  lookupCodes: [
+    { name: 'agencyVal', id: '1', isDisabled: false, type: API.AGENCY_CODE_SET_NAME },
+    { name: 'roleVal', id: '1', isDisabled: false, type: API.ROLE_CODE_SET_NAME },
+  ] as ILookupCode[],
+};
+
+const store = mockStore({
+  [reducerTypes.LOOKUP_CODE]: lCodes,
+});
+
+const form = (
+  <Formik initialValues={{ projectNumber: 'test', assessed: '' }} onSubmit={noop}>
+    {(props: any) => (
+      <Provider store={store}>
+        <BuildingReviewPage
+          classifications={[]}
+          agencies={[]}
+          occupantTypes={[]}
+          predominateUses={[]}
+          constructionType={[]}
+        />
+      </Provider>
+    )}
+  </Formik>
+);
+
+it('renders correctly', () => {
+  const { container } = render(form);
+  expect(pretty(container.innerHTML)).toMatchSnapshot();
+});
+
+describe('identification section', () => {});
+
+it('building identificaiton fields disabled by default', () => {
```

Comment:
Tried placing these in a describe block and using shared query selectors - but it was keeping the initial state of when it queried the element before clicking the edit button